README.md

```diff
@@ -61,13 +61,13 @@ It is still early days for the Obol Network and things are under active developm
 We are moving fast so check back in regularly to track the progress.
 
 Charon is a distributed validator, so its main responsibility is performing validation duties.
```

Comment:
we can also say what are the blockers for integrating prysm, nimbus or vouch.

Reply:
There's a table for that above these changes :)